eluvio

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+node $SOURCE_DIR/utilities/EluvioCli.js $* 
```

Comment:
How about `elv-admin`?  (`eluvio` is too generic and we have been avoiding using the full name in tools)

Reply:
changed to elv-admin.
Though eluvio would be an easy to remember and type "core" tool, but I'm ok with elv-admin.